help/compute/diff-studio.md

```diff
@@ -353,7 +376,42 @@ The export feature provides an extension of your project with [scripting](script
 * non-elementary and special functions' use
 * Datagrok packages' functions call
 
+## Solver management
```

Comment:
Probably "Solver tuning" or "Solver settings" would be better?

Reply:
Replaced by "Solver settings"